SFS_config.m

```diff
@@ -352,20 +352,20 @@
 % Maximum modal order aka. spatial bandwidth of desired sound field. If left
 % empty, the value is set by nfchoa_order(), which may suboptimal depending on
 % the geometry, e.g. number of secondary sources and shape of the secondary 
-% source distribution
-conf.localwfs_sbl.order = [];
+% source distribution.
+conf.localwfs_sbl.order = []; % integer
 % Due to stability issues for the time-domain implementation of synthesis 
 % of a point source, conventional WFS has to be used for the low frequencies. 
 % fc defines the crossover frequency between the WFS and LWFS-SBL. If left
 % empty, this frequency is estimated by aliasing_frequency().
-conf.localwfs_sbl.fc = [];
+conf.localwfs_sbl.fc = []; % float
```

Comment:
I added the `% float` comment here. Is this ok, or has it to be an integer value for the crossover frequency?